pages/common/ag.md

```diff
@@ -2,26 +2,26 @@
 
 > The Silver Searcher. Like ack, but faster.
 
-- Find files containing "foo", and print the line matches in context:
+- Find files containing "foo", and print the line matches in context (`-F` is an alias for `--literal`):
```

Comment:
What does `--literal` do?

Reply:
@sbrl As per the documentation `Don't parse PATTERN as a regular expression`

Reply:
I would like to keep the first example without any flags. We should change the description instead.

Reply:
Also, maybe bring up the penultimate example to the top. I think that might clarify that foo may be a normal text as well as a regular expression.